### DISCUSSION TITLE
Constraint reference image

### DISCUSSION THREAD

**Discussion by LeXXik** [jrouwe/JoltPhysics.js#67] [Q&A] [answered]
The constraint docs sometimes mention to refer to image for reference. Where can I find that image?

https://jrouwe.github.io/JoltPhysics/class_swing_twist_constraint_settings.html#a4f47f4487992a1af9da193d784a9a677

**Comment by jrouwe** [comment, marked as answer]
It just refers to the image at the top of the page:

![image](https://github.com/jrouwe/JoltPhysics.js/assets/1621693/c67f45dd-6e5c-49dc-9d7d-bf501fd8ae7e)

I've updated the text to make that more clear.
